hyperspy/misc/date_time_tools.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Copyright 2007-2016 The HyperSpy developers
+#
+# This file is part of  HyperSpy.
+#
+#  HyperSpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+#  HyperSpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
+
+import datetime
+from dateutil import tz, parser
+
+
+def metadata_to_datetime(metadata):
```

Comment:
There is a bit of asymmetry in these functions now. Maybe make this `ISO_format_to_datetime()`, with an optional time zone field? Then this could be used in `ISO_format_to_serial_date()` below. If not, then why not have a `datetime_to_metadata()` function?